changelog.d/20240614_180451_danyal.faheem_upgrade_mysql_utf8mb4.md

```diff
@@ -0,0 +1 @@
+- 💥[Feature] Upgrade default charset and collation of mysql to utf8mb4 and utf8mb4_unicode_ci respectively (by @Danyal-Faheem)
```

Comment:
We can mention that if the operator is running MySQL on a dedicated server (RUN_MYSQL=False), they are responsible for doing this change manually on their system/instance.

Reply:
Added this small note in the changelog entry.